doc/install.rst

```diff
@@ -46,15 +46,13 @@ Start by looking at the tutorials on our sidebar, good luck!
 Which Python?
 -------------
 
-PyGMT is tested to run on **Python 3.7 or greater**. Older Python versions may
-work, but there is no guarantee that PyGMT will behave as expected.
+PyGMT is tested to run on **Python 3.7 or greater**.
 
 We recommend using the `Anaconda <https://www.anaconda.com/distribution>`__
 Python distribution to ensure you have all dependencies installed and the
 `conda <https://docs.conda.io/projects/conda/en/latest/>`__
-package manager is available. Installing Anaconda does not require administrative
-rights to your computer and doesn't interfere with any other Python
-installations on your system.
+package manager is available. Installing Anaconda does not require administrative rights
+to your computer and won't interfere with any other Python installations on your system.
```

Comment:
What's the reason for the changes here? Most lines in `doc/install.rst` are less than 80 characters.

Reply:
Ok, I'll revert the change.